contexts/authContext.tsx

```diff
@@ -48,15 +53,15 @@ function AuthProvider({ children }: IProps) {
       response.authentication = null; //find better way to remove response
       setUserDetails({ email: null, accessToken: null });
     } catch (error: any) {
-      console.log('logout error', error);
       Alert.alert(
         'Error',
         'There was a problem logging out. Please try again.'
       );
     }
   }
 
-  async function login() {
+  async function googleLogin() {
+    console.log('login()');
```

Comment:
Since you've fixed the login/out bugs (thanks for that BTW), we can probably remove this console log.

Reply:
changed alongside other console.logs